presto-main/src/test/java/com/facebook/presto/operator/scalar/TestStringFunctions.java

```diff
@@ -1045,4 +1045,26 @@ public void testCharConcat()
 
         assertFunction("concat(cast(null as char(1)), cast(' ' as char(1)))", createCharType(2), null);
     }
+
+    @Test
+    public void testStartsWith()
+    {
+        assertFunction("starts_with('abcd', 'ab')", BOOLEAN, true);
+        assertFunction("starts_with('abcd', '')", BOOLEAN, true);
+        assertFunction("starts_with('abcd', 'ba')", BOOLEAN, false);
+        assertFunction("starts_with('', 'ba')", BOOLEAN, false);
+        assertFunction("starts_with(NULL, 'ba')", BOOLEAN, null);
+        assertFunction("starts_with('abcd', NULL)", BOOLEAN, null);
```

Comment:
LGTM, can you add a few more test cases like
(NULL, NULL)
('',  NULL)
(NULL, '')

Similar for the endswith tests.